srcpkgs/pinephone-kernel/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'pinephone-kernel'
 pkgname=pinephone-kernel
-version=5.8.10
+version=5.10.12
 revision=1
-_commit=b8647e317c32f64543c32465abb7b4651e6e3199
-wrksrc="linux-${_commit}"
+_tag=orange-pi-5.10-20210130-2109
```

Comment:
Why is it this weird tag?

Reply:
The author uses an `orange-pi-{kernver}` branch to release all their embedded patches, including the pinephone. This tag references the latest release on the 5.10 branch. The `orange-pi-...` branch recommended for use by the author https://xnux.eu/devices/pine64-pinephone.html#toc-how-my-kernel-tree-is-updated